spec/02-integration/09-hybrid_mode/09-config-compat_spec.lua

```diff
@@ -125,29 +129,6 @@ describe("CP/DP config compat transformations #" .. strategy, function()
           -- ]]
         },
       }
-
-      cors = admin.plugins:insert {
```

Comment:
care to explain why this is gone?

Reply:
Oh, the git diff is a bit confusing. That was just some refactoring effort but I'm very open to discussion if it actually makes sense.  What I noticed is that there were a few things that could get tidied up. 

In line: [#111](https://github.com/Kong/kong/blob/3.5.0/spec/02-integration/09-hybrid_mode/09-config-compat_spec.lua#L111)
We were defining something for all tests cases:
```lua
local rate_limit, cors, opentelemetry, zipkin
```

But:
- `rate_limit` was then created in this `lazy_setup` and then cleaned up in `lazy_teardown`. The respective test cases just did assertions
- `cors` worked similarly but was not cleaned up in `lazy_teardown`
- `opentelemetry` and `zipkin` were created before assertions and cleaned up at the end of tests cases

So I had two options - use the first convention 1️⃣  (`rate_limit` and `cors` style) which creates everything in `lazy_setup` and the test case just verifies assertion or use 2️⃣  `opentelemetry` and `zipkin` style that sets up everything before assertions and cleans up after them. 

I decided to go with option :two: as it also allowed me to narrow down the amount of variables available in the scope and define the variable next to plugin creation. Instead of defining everything in line 111 and then reassigning them in test cases